### PR TITLE
corner radius changed from default to optional

### PR DESCRIPTION
**Pull request by okarakose**
content layout corner radius was defined as a default value. it is changed to optional value.

    public EasyDialog setCornerRadius(int cornerRadius)  // cornerRadius parameter should be pixel value

### CODE REVIEW

File: easydialog/src/main/java/com/michael/easydialog/EasyDialog.java [hunk 1]
```diff
@@ -402,13 +409,18 @@ public EasyDialog show()
             }
 
             if (cornerRadius > 0) {
```
Comment (michaelye) [inline easydialog/src/main/java/com/michael/easydialog/EasyDialog.java]: I think the code style should change from
```
if(xxx){

}else{

}
```

to
```

if(xxx)
{

}
else
{

}
```

**Review by michaelye — COMMENTED** [review]
Thanks for the feature.Great job! But please change the code style to meet the original code style.Then I would merge this PR.

**Comment by okarakose** [comment]
okay, I have done it

**Comment by michaelye** [comment]
@okarakose Good job!Thanks for the PR.

**Comment by okarakose** [comment]
could you publish new version to use this feature ?

**Pull request merged**